datahub-web-react/src/app/entity/dataProcessInstance/DataProcessInstanceEntity.tsx

```diff
@@ -168,6 +151,10 @@ export class DataProcessInstanceEntity implements Entity<DataProcessInstance> {
     renderPreview = (_: PreviewType, data: DataProcessInstance) => {
         const genericProperties = this.getGenericEntityProperties(data);
         const parentEntities = getParentEntities(data);
+        type PreviewEntity = {
```

Comment:
what is going on here? 

why is PreviewEntity type defined in multiple places here.. Can we define in one place?